example/android/app/build.gradle

```diff
@@ -137,10 +136,10 @@ android {
 }
 
 dependencies {
-    compile project(':react-native-share')
-    compile fileTree(dir: "libs", include: ["*.jar"])
-    compile "com.android.support:appcompat-v7:23.0.1"
-    compile "com.facebook.react:react-native:+"  // From node_modules
+    implementation project(':react-native-share')
+    implementation fileTree(dir: "libs", include: ["*.jar"])
+    implementation "com.android.support:appcompat-v7:28.0.0"
+    implementation "com.facebook.react:react-native:+"  // From node_modules
```

Comment:
this should be compileOnly

Reply:
I'm looking into this - I'm more familiar with apps (which use libraries via implementation) then library development such as here, where compileOnly might make sense. I think you're right on first glance but am verifying with docs right now.

Reply:
...and found the answer right after. Yes, compileOnly is better as client apps will have this declared. I'll alter this

Reply:
...then contradicted myself with testing. I just pushed two changes after verifying what's necessary for the example to work out of the box. This comment was on the example app, so it must have these as implementation as it needs them at runtime - in the library (the parent of the example) compileOnly could work, but this doesn't run unless they are implementation.

The other change was that the relative path I attempted in the package.json of the example - a reference to '..' for the necessary react-native-share dependency resulted in the bundler becoming confused about where to resolve things. The original reference to some other github repo for the react-native-share dep didn't seem right, so I put in the official repo and I successfully run the example out of the box now